### PR TITLE
NN-3164 Update circle orb and node versions to fix security audit

### PR DESCRIPTION
**Pull request by jsrobertson**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -73,8 +73,10 @@ jobs:
   cypress_integration_tests:
     working_directory: ~/app
     docker:
-      - image: circleci/node:14-buster-browsers
-      - image: circleci/redis:buster
+      - image: cimg/node:14.15.3-browsers
```
Comment (petergphillips) [inline .circleci/config.yml:76]: think this should be running the latest 14 version, same as the Dockerfile

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@2.2
+  hmpps: ministryofjustice/hmpps@2.3.0
```
Comment (mattops) [inline .circleci/config.yml:4]: should be 2.3

File: Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM node:14-buster
+FROM node:14.15-buster
```
Comment (petergphillips) [inline Dockerfile:1]: this docker file is really out of date - could do with updating to match the others at some point

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**